apps/condo/domains/organization/constants/common.js

```diff
@@ -1,5 +1,14 @@
+const {
+    ORGANIZATION_TICKET_VISIBILITY,
+    PROPERTY_TICKET_VISIBILITY,
+    PROPERTY_AND_SPECIALIZATION_VISIBILITY,
+    ASSIGNED_TICKET_VISIBILITY,
+    TICKET_VISIBILITY_OPTIONS,
+} = require('@condo/domains/scope/constants')
```

Comment:
cross-domain link! you can't use scope inside the organization domain. You need to move it inside the organization

Reply:
moved